node-template/src/cli.rs

```diff
@@ -66,6 +66,11 @@ fn run_until_exit<T, C, E>(
 
 	let _ = runtime.block_on(e.into_exit());
 	exit_send.fire();
+
+	// we eagerly drop the service so that the internal exit future is fired,
+	// but we need to keep holding a reference to the global telemetry guard
+	let _telemetry = service.telemetry();
```

Comment:
@andresilva can we not fix this in a different way? This feels slightly awkward, when all people need to copy paste this fix?